packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -53,6 +59,17 @@ export function Header({
     [setLayout, layout]
   );
 
+  useEffect(() => {
+    const paramTimeZone = searchParams?.get("timezone");
+    if (paramTimeZone) {
+      setTimezone(paramTimeZone);
+    } else {
```

Comment:
There is no error handling here in case the passed in timezone is invalid. This would result in the breaking of the public booking page.